css/style.css

```diff
@@ -1,6 +1,881 @@
+body {
+	margin: 0;
+	box-sizing: border-box;
+}
+
+.conteiner {
+	max-width: 1030px;
+	width: 100%;
+	margin: 0 auto;
+	box-sizing: border-box;
+
+}
+
+ a {
+ 	display: inline-block;
+ 	text-decoration: none;
+ }
+
+ .header {
+ 	width: 100%;
+ 	top: 0;
+ 	left: 0;
+ 	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	height: 116px;
+ }
+
+.header__column {
+	flex: 0 0 55px;
+	height: 55px;
+}
+
+.header__logo {
+	flex: 0 0 55px;
+}
+
+.header__logo_img {
+	max-width: 100%;
+	display: block;
+}
+
+.header__logo_tabimg {
+	display: none;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5730635)